docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -38,41 +45,55 @@ If you are using Docker, use an environment variable to enable public dashboards
 --env GF_FEATURE_TOGGLES_ENABLE=publicDashboards
 ```
 
-> **Note:** For Grafana Cloud, contact support to have the feature enabled.
+{{% admonition type="Note" %}}
+
+For Grafana Cloud, contact support to have the feature enabled.
```

Comment:
The enablement of feature toggles by support is only available for paid customers (starting with Grafana Pro). Maybe we want to add this limitation?